Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -968,7 +968,8 @@ class AEPEdgeFunctionalTests: TestBase, AnyCodableAsserts {
         mockNetworkService.setExpectationForNetworkRequest(url: TestConstants.EX_EDGE_INTERACT_PROD_URL_STR, httpMethod: HttpMethod.post, expectedCount: 1)
         mockNetworkService.setMockResponse(url: TestConstants.EX_EDGE_INTERACT_PROD_URL_STR, httpMethod: HttpMethod.post, responseConnection: httpConnection)
 
-        mockNetworkService.assertAllNetworkRequestExpectations()
+        // Default retry time period is 5 sec - provide extra timeout buffer
+        mockNetworkService.assertAllNetworkRequestExpectations(timeout: 10)
```

Comment:
Let's extract timeouts as constants.

Reply:
Also do we have any retry scenarios that require the retry timeout to be considered or this is just adding to account for retry in case of unexpected failure?